suite/tests/client-interface/drwrap-drreg-test.dll.c

```diff
@@ -331,7 +351,11 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
     /* Look for nop;nop;nop in reg_val_test() and 4 nops in multipath_test(). */
     if (instr_is_app(inst)) {
         int *nop_count = (int *)user_data;
+#if defined(RISCV64)
+        if (instr_is_nop(inst)) {
+#else
         if (instr_get_opcode(inst) == OP_nop IF_ARM(|| instr_is_mov_nop(inst))) {
+#endif
```

Comment:
Probably originally because this is recognizing a specific opcode we control in the app and we don't want a broad check that recognizes other nop variants (on x86 there are several encoding variants of different sizes all considered nops).  The tradeoff of a possible false positive pattern recognition might be outweighed by simpler cross-arch code so probably ok to change it.

Reply:
Done